integreat_compass/locale/de/LC_MESSAGES/django.po

```diff
@@ -448,6 +471,45 @@ msgstr "Passwort eingeben"
 msgid "Sign In"
 msgstr "Einloggen"
 
+#: cms/templates/authentication/registration.html
+msgid "Registration"
+msgstr "Registrierung"
+
+#: cms/templates/authentication/registration.html
+msgid "Register as an Offer Provider"
+msgstr "Als Maßnahmen-Anbieter registrieren"
+
+#: cms/templates/authentication/registration.html
+msgid "Sign Up"
+msgstr "Registrieren"
+
+#: cms/templates/emails/activation_email.html
+#: cms/templates/emails/activation_email.txt
+msgid "Welcome"
+msgstr "Willkommen"
+
+#: cms/templates/emails/activation_email.html
+#: cms/templates/emails/activation_email.txt
+msgid ""
+"You are receiving this e-mail because you created an account with Integreat "
+"Compass."
+msgstr ""
+"Sie erhalten diese Email da Sie einen Account im Integreat Compass angelegt "
+"haben."
+
+#: cms/templates/emails/activation_email.html
+#: cms/templates/emails/activation_email.txt
+msgid "Please click the link below to activate your account."
```

Comment:
The tech writer in my would write `Please use the link below` to avoid being to specific about the input device. It would also work with a keyboard, etc.